x-pack/plugins/synthetics/public/apps/synthetics/components/monitors_page/overview/overview/quick_filters.tsx

```diff
@@ -6,36 +6,49 @@
  */
 import React from 'react';
 import { i18n } from '@kbn/i18n';
-import { EuiFilterGroup, EuiFilterButton } from '@elastic/eui';
+import { EuiButtonGroup } from '@elastic/eui';
+import { useOverviewStatus } from '../../hooks/use_overview_status';
 import { useGetUrlParams, useUrlParams } from '../../../../hooks';
 
 export const QuickFilters = () => {
   const { statusFilter } = useGetUrlParams();
   const [_, updateUrlParams] = useUrlParams();
+  const { status } = useOverviewStatus({ scopeStatusByLocation: true });
 
   const handleFilterUpdate = (monitorStatus: string) => {
-    return () => {
-      updateUrlParams({ statusFilter: statusFilter !== monitorStatus ? monitorStatus : undefined });
-    };
+    updateUrlParams({ statusFilter: statusFilter !== monitorStatus ? monitorStatus : undefined });
   };
+
+  const statusButtons = [
+    {
+      id: 'up',
+      label: UP_LABEL,
+    },
+    {
+      id: 'down',
+      label: DOWN_LABEL,
+    },
+    {
+      id: 'disabled',
+      label: DISABLED_LABEL,
+    },
+  ];
+  if (status?.pending && status?.pending > 0) {
```

Comment:
```suggestion
  if (status?.pending > 0) {
```

Reply:
actually that won't work since pending can also be undefined so it's comparison will throw type error with a number